src/Action/AddAction.php

```diff
@@ -50,10 +51,8 @@ class AddAction extends BaseAction
         'viewVar' => null,
         'relatedModels' => true,
         'entityKey' => 'entity',
-        'saveOptions' => [
-            'validate' => true,
-            'atomic' => true
-        ],
+        'entityOptions' => [],
+        'saveOptions' => [],
```

Comment:
@lorenzo Oh that's nice. If the same options can be used for both then I too am in favor of not adding a separate key for it. 

@jippi The list of config keys it reasonably long already. If adding new keys can be avoided we should.